test/clt-tests/migration-es-ms/docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM docker:dind
```

Comment:
We already have the same dockerfile with the same content [here](https://github.com/manticoresoftware/manticoresearch/blob/master/test/clt-tests/mysqldump/versions/docker/Dockerfile) 

Use this one. Or expose it in some common (and clear for understanding, like `tests/clt-tests/base_docker`) place and update both scripts (I prefer this way)

Reply:
done